src/LitePsmJob.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import {IJob} from "./interfaces/IJob.sol";
+
+interface SequencerLike {
+    function isMaster(bytes32 network) external view returns (bool);
+}
+
+interface LitePsmLike {
+    function chug() external returns (uint256 wad);
+    function cut() external view returns (uint256 wad);
+    function fill() external returns (uint256 wad);
+    function gush() external view returns (uint256 wad);
+    function rush() external view returns (uint256 wad);
+    function trim() external returns (uint256 wad);
+}
+
+/// @title Call flap when possible
+contract LitePsmJob is IJob {
+    SequencerLike public immutable sequencer;
+    address public immutable litePsm;
```

Comment:
The actual storage variable can have the `LitePsmLike` type, so you don't need to use casting everytime you need to use it.